cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -125,10 +127,6 @@ match_types! {
 		MultiLocation { parents: 1, interior: Here } |
 		MultiLocation { parents: 1, interior: X1(Plurality { .. }) }
 	};
-	pub type ParentOrSiblings: impl Contains<MultiLocation> = {
-		MultiLocation { parents: 1, interior: Here } |
-		MultiLocation { parents: 1, interior: X1(_) }
```

Comment:
Not an XCM expert, but what does a sibling "anything" here include other than parachains? Accounts?

Reply:
Yes, it could be anything in the parent, so not really a sibling. Accounts in the parent, pallets, governance bodies, for example.